libraries/AP_HAL_ChibiOS/hwdef/iomcu/hwdef.inc

```diff
@@ -5,7 +5,7 @@
 FLASH_RESERVE_START_KB 4
 
 # board ID. See Tools/AP_Bootloader/board_types.txt
-APJ_BOARD_ID 3
+APJ_BOARD_ID EXT_HW_RADIOLINK_MINI_PIX
```

Comment:
We definitely won't be enforcing this value anywhere.

We should probably fix it to be an *actual*, *distinct* board ID, potentially also including different MCU types as different boards (and fixing CubeOrangeSecondary-IO to not use the same ID as an f103-based IOMCU!

Reply:
Yeah probably need a different value for different MCUs in IOMCU

Reply:
> Yeah probably need a different value for different MCUs in IOMCU

I think I've allocated and used sufficient now.